bn-apps/cordapp-updates-distribution/corda-updates-app/src/main/kotlin/net/corda/cordaupdates/app/member/CordaUpdatesSerialisationWhitelist.kt

```diff
@@ -0,0 +1,9 @@
+package net.corda.cordaupdates.app.member
+
+import net.corda.businessnetworks.cordaupdates.core.SyncerConfiguration
+import net.corda.businessnetworks.cordaupdates.core.CordappSource
+import net.corda.core.serialization.SerializationWhitelist
+
+class CordaUpdatesSerialisationWhitelist : SerializationWhitelist {
```

Comment:
Comments comments comments please :) Why is this needed?

Reply:
It wasn't actually needed anymore. Removed the class